### ISSUE TITLE
suggestion

### ISSUE THREAD

**Issue by kassasin** [jupyter-robotics/jupyterlab-urdf#58]
it is better if it can visualize robot frame

**Comment by kassasin** [comment]
@IsabelParedes Yes,because when i write urdf file,i must know previous frame before i write joint rotation angle in order to connect next frame correctly.So,before i meet jupyterlab-urdf,I use rviz to visualize robot and write urdf .It is inconvenient because I must restart rviz after change urdf .Does jupyterlab-urdf can visualize link frame?

**Comment by IsabelParedes** [comment]
Hi @kassasin ! Currently, the extension doesn't support frames, but this is a great idea. I'll look into it a bit more, but I think it should be doable.
